.github/workflows/call-bump-pr-workflow.yml

```diff
@@ -0,0 +1,44 @@
+# Open a PR to bump the version of semgrep in semgrep-app, semgrep-rpc, or semgrep-action.
+# This is called by start-release.yml once validate-release.yml has finished.
+
+name: call-bump-pr-workflow
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+      repository:
+        required: true
+        type: string
+
+jobs:
+  bump-version:
+    name: Bump semgrep version
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Get JWT for semgrep-ci GitHub App
+        id: jwt
+        uses: docker://public.ecr.aws/y9k7q4m1/devops/cicd:latest
+        env:
+          EXPIRATION: 600 # seconds
+          ISSUER: ${{ secrets.SEMGREP_CI_APP_ID }} # semgrep-ci GitHub App id
+          PRIVATE_KEY: ${{ secrets.SEMGREP_CI_APP_KEY }}
+      - name: Get token for semgrep-ci GitHub App
+        id: token
+        run: |
+          TOKEN="$(curl -X POST \
+          -H "Authorization: Bearer ${{ steps.jwt.outputs.jwt }}" \
+          -H "Accept: application/vnd.github.v3+json" \
+          "https://api.github.com/app/installations/${{ secrets.SEMGREP_CI_APP_INSTALLATION_ID }}/access_tokens" | \
+          jq -r .token)"
+          echo "::add-mask::$TOKEN"
+          echo "::set-output name=token::$TOKEN"
+      - name: Invoke workflow
+        env:
+          GITHUB_TOKEN: "${{ steps.token.outputs.token }}"
+          NEW_SEMGREP_VERSION: "${{ inputs.version }}"
+          REPOSITORY: "${{ inputs.repository }}"
+        run: |
+          gh workflow run bump_version.yaml --repo "${REPOSITORY}" --raw-field version="${NEW_SEMGREP_VERSION}"
```

Comment:
Do you know if this waits for completion of that workflow? I don't think it matters either way but a comment would be nice so that's it's clearer for follow on.

Reply:
It does not wait for completion. Added a comment